<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemExportSettingDialog</class>
 <widget class="QDialog" name="ItemExportSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="name">
       <property name="text">
        <string/>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;ID:</string>
       </property>
       <property name="buddy">
        <cstring>id</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="id"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="typeEmbed">
         <property name="text">
          <string>&amp;Embed</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="typeMerge">
         <property name="text">
          <string>&amp;Merge to </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="merge"/>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="typeCustom">
         <property name="text">
          <string>&amp;Custom</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="custom"/>
         </item>
         <item>
          <widget class="QComboBox" name="customBase"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="typeNative">
         <property name="text">
          <string>&amp;Native</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="nativeBase"/>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="typeSkip">
         <property name="text">
          <string>&amp;Skip</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="embedWithTouch">
         <property name="text">
          <string>with &amp;Touch</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Properties:</string>
       </property>
       <property name="buddy">
        <cstring>visible</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QWidget" name="properties" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="visible">
          <property name="text">
           <string>&amp;Visible</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="position">
          <property name="text">
           <string>&amp;Position</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="size">
          <property name="text">
           <string>Si&amp;ze</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="color">
          <property name="text">
           <string>Co&amp;lor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="text">
          <property name="text">
           <string>Te&amp;xt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="image">
          <property name="text">
           <string>&amp;Image</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>87</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>id</tabstop>
  <tabstop>typeEmbed</tabstop>
  <tabstop>embedWithTouch</tabstop>
  <tabstop>typeMerge</tabstop>
  <tabstop>merge</tabstop>
  <tabstop>typeCustom</tabstop>
  <tabstop>custom</tabstop>
  <tabstop>customBase</tabstop>
  <tabstop>typeNative</tabstop>
  <tabstop>nativeBase</tabstop>
  <tabstop>typeSkip</tabstop>
  <tabstop>visible</tabstop>
  <tabstop>position</tabstop>
  <tabstop>size</tabstop>
  <tabstop>color</tabstop>
  <tabstop>text</tabstop>
  <tabstop>image</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemExportSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemExportSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
